persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -69,5 +67,8 @@ private PersistToolsConstants() {
     public static final String PERSIST_TOML_FILE = "Persist.toml";
     public static final String DATABASE_CONFIGURATION_BAL = "database_configuration.bal";
 
-
+    public static final String BALLERINA_MYSQL_DRIVER = "ballerinax/mysql.driver";
+    public static final String JAVA_11 = "java11";
+    public static final String PATH = "path";
```

Comment:
This is used when retrieving the path of the MySQL driver jar dependency[[1]](https://github.com/ballerina-platform/persist-tools/blob/0c76bc1ef9fa5c79dd79d68e85b7c791e0e390df/persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java#L265)

[1] https://github.com/ballerina-platform/persist-tools/blob/0c76bc1ef9fa5c79dd79d68e85b7c791e0e390df/persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java#L265

Reply:
Seems like it is the key to the path from dependency properties map.

How about naming it "PROPERTY_KEY_PATH" / "DEP_PROPERTY_KEY_PATH"? WDYT? IMO it would be much clearer.

Reply:
Updated in [f568f12](https://github.com/ballerina-platform/persist-tools/pull/53/commits/f568f12dead6d22dfecd502349fd2ff4b86395f1)